tests/foreman/ui/test_adusergroups.py

```diff
@@ -17,7 +17,26 @@ def test_create_ldap_auth_withAD(self):
         1. Create a new LDAP Auth source with AD
         2. Fill in all the fields appropriately for AD
 
-        @Assert: Whether creating LDAP Auth with AD is successful
+        @Assert: Whether creating LDAP Auth with AD is successful.
+
+        @status: Manual
```

Comment:
Saw that you have changed from `status` to `Status` in this PR, maybe you may want to do the same here.